mlflow/langchain/__init__.py

```diff
@@ -607,7 +607,9 @@ def predict(
         from mlflow.langchain.api_request_parallel_processor import process_api_requests
 
         messages, return_first_element = self._prepare_messages(data)
-        results = process_api_requests(lc_model=self.lc_model, requests=messages)
+        results = process_api_requests(
+            lc_model=self.lc_model, requests=messages, configurables=params
```

Comment:
Could you update the docstring of `params` argument to mention that it will passed as `config={"configurable": params}` to the underlying chain?

Reply:
Good point, done